docs/docs/KerasStyleAPIGuide/Layers/advancedActivations.md

```diff
@@ -0,0 +1,332 @@
+---
+## **ELU**
+Exponential Linear Unit.
+
+It follows: f(x) =  alpha * (exp(x) - 1.) for x < 0, f(x) = x for x >= 0.
+
+**Scala:**
+```scala
+ELU(alpha = 1.0, inputShape = null)
+```
+**Python:**
+```python
+ELU(alpha=1.0, input_shape=None)
+```
+
+**Parameters:**
+
+* `alpha`: Double, scale for the negative factor.
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
+
+**Scala example:**
+```scala
+import com.intel.analytics.bigdl.nn.keras.{Sequential, ELU}
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor._
+
+val model = Sequential[Float]()
+model.add(ELU(inputShape = Shape(3)))
+val input = Tensor[Float](2, 3).randn()
+val output = model.forward(input)
+```
+Input is:
+```scala
+input: com.intel.analytics.bigdl.tensor.Tensor[Float] =
+-0.13405465	0.05160992	-1.4711418
+1.5808829	-1.3145303	0.6709266
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x3]
+```
+Output is:
+```scala
+output: com.intel.analytics.bigdl.nn.abstractnn.Activity =
+-0.1254577	0.05160992	-0.77033687
+1.5808829	-0.73139954	0.6709266
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x3]
+```
+
+**Python example:**
+```python
+import numpy as np
+from bigdl.nn.keras.topology import Sequential
+from bigdl.nn.keras.layer import ELU
+
+model = Sequential()
+model.add(ELU(1.2, input_shape=(3, )))
+input = np.random.random([2, 3])
+output = model.forward(input)
+```
+Input is:
+```python
+[[0.98624221 0.93625254 0.51908689]
+ [0.93353209 0.72231469 0.41468629]]
+```
+Output is
+```python
+[[0.98624223 0.93625253 0.5190869 ]
+ [0.93353206 0.7223147  0.4146863 ]]
+```
+
+---
+## **LeakyReLU**
+Leaky version of a Rectified Linear Unit.
+
+It allows a small gradient when the unit is not active: f(x) = alpha * x for x < 0, f(x) = x for x >= 0.
+
+**Scala:**
+```scala
+LeakyReLU(alpha = 0.01, inputShape = null)
```

Comment:
use default value 0.3.. also for python

Reply:
The default value is 0.01?

Reply:
Default value should refer to Keras is 0.3 please double check keras doc.

Reply:
Update and rerun the code to get results.